app/serializers/event_hour_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Defines EventHour attributes to be returned in JSON
+class EventHourSerializer < ApplicationSerializer
+  attributes :event_hour_id, :start_time, :end_time, :open_slots
+
+  def open_slots
+    object&.open_slots
```

Comment:
Why would the serializer `object` ever be nil?

Reply:
Yes, serializer object won't return nil, rubocop made me to do that.
i'll update that to,
  attribute :open_slots do
    object.open_slots
  end

this should work fine.